### PR TITLE
Section splitting

### PR DESCRIPTION
**Pull request by susifohn**

### CODE REVIEW

**Comment by susifohn** [comment]
From Christian Kissling

File: scrc/preprocessors/extractors/spider_specific/section_splitting_functions.py [hunk 1]
```diff
@@ -25,6 +25,87 @@ def XX_SPIDER(decision: Union[bs4.BeautifulSoup, str], namespace: dict) -> Optio
...
+        # remove spaces between two line breaks, watch the space before +!!
+        decision = re.sub('\\n +\\n', '\\n\\n', decision,0, re.MULTILINE)
+        # split the lines when there are two line breaks
+        lines = decision.split('\n\n')
+        for element in lines:
+            element = element.replace('  ',' ')
+            paragraph = clean_text(element)
+            if paragraph not in ['', ' ', None]:  # discard empty paragraphs
+                paragraphs.append(paragraph)
+        return paragraphs
```
Comment (JoelNiklaus) [inline scrc/preprocessors/extractors/spider_specific/section_splitting_functions.py:61]: Try to refactor this code to avoid code duplication

**Review by JoelNiklaus — CHANGES_REQUESTED** [review]

File: scrc/preprocessors/extractors/spider_specific/section_splitting_functions.py (hunk 1 above)
Comment (susifohn) [reply scrc/preprocessors/extractors/spider_specific/section_splitting_functions.py]: Worked an other approach for section splitting. See #16

**Review by susifohn — COMMENTED** [review]
I tried a different approach to section splitting that I think works well. At least for _UR_Gerichte_

**Comment by JoelNiklaus** [comment]
Looks good, thanks.

**Pull request merged**